CMakeLists.txt

```
@@ -310,6 +310,13 @@ elseif(UNIX)
       # The FindOSS.cmake module does not need to be installed like the JACK modules because it
       # does not link any library; it only adds an include directory and compile definition.
     endif()
+
+    CHECK_INCLUDE_FILE(sys/audioio.h HAVE_SYS_AUDIOIO_H)
```

```suggestion
    check_include_file(sys/audioio.h HAVE_SYS_AUDIOIO_H)
```
Use lowercase for consistency.